tests/features/comment_test.py

```diff
@@ -1,7 +1,20 @@
+import pytest
+
 from testing.runner import and_exit
 from testing.runner import trigger_command_mode
 
 
+@pytest.fixture
+def three_lines_with_indentation(tmpdir):
+    f = tmpdir.join('f')
+    f.write(
+        '\n'.join(
+            f'line_{i}' if i < 1 else f'    line_{i}' for i in range(3)
+        ),
+    )
```

Comment:
this is way too clever, just write the contents literally

especially in test code the thing should be obvious to read at a glance and this requires a brain-interpreter to figure out what it's doing

Reply:
Yes, i understand, fixed.